machetli/pddl/files.py

```diff
@@ -73,8 +74,8 @@ def temporary_files(state: dict) -> tuple:
     problem_file = tempfile.NamedTemporaryFile(
         mode="w+t", suffix=".pddl", delete=False)
     problem_file.close()
-    write_files(state, domain_filename=domain_file.name,
-                problem_filename=problem_file.name)
+    write_files(state, domain_path=domain_file.name,
```

Comment:
Does this do the right thing if you give it a domain path like `sub/directory/domain.pddl`? It looks like it would ignore the directory component and just use the name. Also, I would convert the paths to paths in the first lines, now that we are accepting strings as well, otherwise this would crash on `domain_path='domain.pddl'`